src/libraries/System.IO.FileSystem/tests/FileInfo/Open.cs

```diff
@@ -13,7 +13,6 @@ protected override FileStream CreateFileStream(string path, FileMode mode)
             return new FileInfo(path).Open(mode);
         }
 
-        [Theory, MemberData(nameof(StreamSpecifiers))]
```

Comment:
Is it because of these attributes are inherited?

Reply:
Exactly.